pallets/staking-rewards/src/mock.rs

```diff
@@ -87,24 +83,24 @@ construct_runtime!(
         NodeBlock = Block,
         UncheckedExtrinsic = UncheckedExtrinsic,
     {
-        System: system::{Pallet, Call, Config, Storage, Event<T>},
-        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-        Authorship: pallet_authorship::{Pallet, Storage},
-        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
-        StakingRewards: pallet_gear_staking_rewards::{Pallet, Call, Storage, Config<T>, Event<T>},
-        Staking: pallet_staking::{Pallet, Call, Storage, Config<T>, Event<T>},
-        Session: pallet_session::{Pallet, Call, Storage, Config<T>, Event},
-        Historical: pallet_session_historical::{Pallet, Storage},
-        Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>},
-        BagsList: pallet_bags_list::<Instance1>::{Pallet, Event<T>},
-        Sudo: pallet_sudo::{Pallet, Call, Storage, Config<T>, Event<T>},
-        Utility: pallet_utility::{Pallet, Call, Event},
-        ElectionProviderMultiPhase: multi_phase::{Pallet, Call, Event<T>},
+        System: system,
```

Comment:
You've changed order of pallets. Seems that has no effect, but pls verify that these hooks aren't chained between each other

Reply:
The order set the same as in runtime. Tests are valid